pkg/sqlite/load.go

```diff
@@ -172,35 +172,39 @@ func (s *SQLLoader) AddPackageChannels(manifest registry.PackageManifest) error
 
 		channelEntryCSVName := c.CurrentCSVName
 		depth := 1
-		for {
-			rows, err := getReplaces.Query(channelEntryCSVName)
-			if err != nil {
+		rows, err := getReplaces.Query(channelEntryCSVName)
```

Comment:
Removing the outer for loop here means that we only every build update chains of depth 1